src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
         and we will work with you and the appropriate development team to resolve it.
         Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
       </p>
+	  
```

Comment:
As a general comment, whitespace indenting does not seem to match the rest of the source code.

Reply:
I hadn't noticed! Now fixed.